src/tools/perf/perftest.c

```diff
@@ -452,13 +451,13 @@ static ucs_status_t setup_sock_rte_p2p(struct perftest_context *ctx)
                       NULL, NULL);
         }
 
-        ctx->sock_rte_group.sendfd     = sockfd;
-        ctx->sock_rte_group.recvfd     = sockfd;
-        ctx->sock_rte_group.peer       = 0;
-        ctx->sock_rte_group.is_server  = 0;
+        ctx->sock_rte_group.peer      = 0;
+        ctx->sock_rte_group.is_server = 0;
     }
 
-    ctx->sock_rte_group.size = 2;
+    ctx->sock_rte_group.sendfd = sockfd;
```

Comment:
I agree there should be one variable for FD. It's just that need to switch roles of "sockfd" and "connfd" because now the usage does not match variable name.

Reply:
> I agree there should be one variable for FD. It's just that need to switch roles of "sockfd" and "connfd" because now the usage does not match variable name.

@yosefe not sure that I understand what we should do to fix matching variable names and roles